packages/spark-core/components/pagination.js

```diff
@@ -35,23 +35,27 @@ export default () => {
   });
 
   // Loop through all pagination items for each default pagination component
-  defaultPagItems.forEach((defaultPagItem) => {
-    // Initially set aria labels on page load
-    setAriaLabel(defaultPagItem);
-  });
-
-  longPagItems.forEach((longPagItem) => {
-    setAriaLabel(longPagItem);
-    longPagItem.addEventListener('click', () => {
-      setAriaLabel(longPagItem);
+  if (defaultPagItems) {
+    defaultPagItems.forEach((defaultPagItem) => {
```

Comment:
Are the changes in this file because you forgot to rebase your forms stuff and merged stuff into staging or is it some weird git history thing lol?

Reply:
Oh, I see that you added in a check for if the elems exist. Good catch. Though, I think the check should be done on longPaginations and defaultPaginations. Because if the main pagination elements don't exist then we don't need to check if the items inside exist.

Reply:
i just added that cause i was getting forEach is not a function on pages other than the pagination page